oneup/flysystem-bundle/3.0/config/packages/oneup_flysystem.yaml

```diff
@@ -0,0 +1,12 @@
+oneup_flysystem:
+    adapters:
+        my_adapter:
```

Comment:
@Nyholm is this best practice?
what about `default_adapter`? and `default_filesystem?`